app/api/tax.py

```diff
@@ -66,10 +69,33 @@ def before_create_object(self, data, view_kwargs):
 
 
 class TaxDetail(ResourceDetail):
+    """
+     tax detail by id
+    """
+    def before_get_object(self, view_kwargs):
+        if view_kwargs.get('event_identifier'):
```

Comment:
@shubham-padia I opened an issue #3856 to implement similar blocks for list methods. Check Issue description in #3856 .